smarts/core/sensors.py

```diff
@@ -71,6 +71,10 @@ class EgoVehicleObservation(NamedTuple):
     mission: Mission
     linear_velocity: np.ndarray
     angular_velocity: np.ndarray
+    linear_acceleration: np.ndarray
```

Comment:
I'm tempted to suggest we combine linear and angular into one field or velocity, accel, jerk in one field. But this may be a bad temptation :)